environment/src/main/java/com/sequenceiq/environment/exception/mapper/AwsDefaultRegionSelectionExceptionMapper.java

```diff
@@ -0,0 +1,22 @@
+package com.sequenceiq.environment.exception.mapper;
+
+import javax.ws.rs.core.Response;
+
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.cloud.aws.AwsDefaultRegionSelectionFailed;
+
+@Component
+public class AwsDefaultRegionSelectionExceptionMapper extends BaseExceptionMapper<AwsDefaultRegionSelectionFailed> {
+
+    @Override
+    Response.Status getResponseStatus(AwsDefaultRegionSelectionFailed exception) {
+        return Response.Status.FORBIDDEN;
```

Comment:
what is the reason to FORBIDDEN?

Reply:
because with the specified credentials CB is unable to verify regions as it gets Forbidden back from AWS

Reply:
I didn't see anywhere in the code this but ok for me